media/js/firefox/australis/australis-uitour.js

```diff
@@ -177,15 +234,28 @@ if (typeof Mozilla == 'undefined') {
         });
     };
 
-    Mozilla.UITour.showFirefoxAccounts = function() {
-        _sendEvent('showFirefoxAccounts');
+    /**
+     * Request the browser open the Firefox Accounts page.
+     *
+     * @param {Object} extraURLCampaignParams - An object containing additional
```

Comment:
Until we have a single documented place for changes, I would suggest adding this to the bedrock docs, so we don't need to look in two places. I believe the long-term plan is to have docs generated directly from these comments in mozilla central. But until that's in place, I would keep our current source of documentation up to date.